File: dataset_reduzido/avatar/Python/Code/atcoder_ARC062_B.py
s = list ( input ( ) )
num_g , num_p = 0 , 0
score = 0
for x in s :
    if x == 'g' :
        if num_g - num_p > 0 :
            score += 1
            num_p += 1
        else :
            score = score
            num_g += 1
    elif x == 'p' :
        if num_g - num_p > 0 :
            score = score
            num_p += 1
        else :
            score -= 1
            num_g += 1
print ( score )

File: output/LLaMa_repaired_1/avatar/Python/Java/atcoder_ARC062_B.java
import java.util.Scanner;

public class atcoder_ARC062_B {
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);
        String[] input = scanner.nextLine().split(" ");
        int numG = Integer.parseInt(input[0]);
        int numP = Integer.parseInt(input[1]);
        int score = 0;
        score = Math.min(numG, numP) + Math.abs(numG - numP) / 2;
        System.out.println(score);
    }
}